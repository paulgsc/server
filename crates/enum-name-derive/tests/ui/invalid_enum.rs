
use enum_name_derive::EnumFilename;

#[derive(EnumFilename)]
pub struct NotAnEnum {} // ❌ Should fail: EnumFilename only supports enums!

fn main() {}

