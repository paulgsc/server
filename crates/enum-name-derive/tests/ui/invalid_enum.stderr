error: proc-macro derive panicked
 --> tests/ui/invalid_enum.rs:4:10
  |
4 | #[derive(EnumFilename)]
  |          ^^^^^^^^^^^^
  |
  = help: message: EnumFilename can only be derived for enums
